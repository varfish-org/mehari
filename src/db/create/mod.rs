//! Creation of mehari internal databases.

pub mod txs;
