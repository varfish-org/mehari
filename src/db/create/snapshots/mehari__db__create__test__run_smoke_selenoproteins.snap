---
source: src/db/create/mod.rs
expression: "String::from_utf8(buf)?"
---
txDb:
  transcripts:
  - id: NM_004261.5
    geneSymbol: SELENOF
    geneId: HGNC:17705
    tags:
    - MANE_SELECT
    - SELENOPROTEIN
    protein: NP_004252.2
    startCodon: 15
    stopCodon: 513
    genomeAlignments:
    - genomeBuild: GRCH38
      contig: NC_000001.11
      cdsStart: 86863473
      cdsEnd: 86914111
      strand: MINUS
      exons:
      - altStartI: 86862444
        altEndI: 86863605
        ord: 4
        altCdsStartI: 382
        altCdsEndI: 1542
        cigar: 1161M
      - altStartI: 86868052
        altEndI: 86868102
        ord: 3
        altCdsStartI: 332
        altCdsEndI: 381
        cigar: 50M
      - altStartI: 86880661
        altEndI: 86880725
        ord: 2
        altCdsStartI: 268
        altCdsEndI: 331
        cigar: 64M
      - altStartI: 86903280
        altEndI: 86903448
        ord: 1
        altCdsStartI: 100
        altCdsEndI: 267
        cigar: 168M
      - altStartI: 86914027
        altEndI: 86914126
        altCdsStartI: 1
        altCdsEndI: 99
        cigar: 99M
  - id: NM_203341.3
    geneSymbol: SELENOF
    geneId: HGNC:17705
    tags:
    - SELENOPROTEIN
    protein: NP_976086.1
    startCodon: 15
    stopCodon: 390
    genomeAlignments:
    - genomeBuild: GRCH38
      contig: NC_000001.11
      cdsStart: 86863546
      cdsEnd: 86914111
      strand: MINUS
      exons:
      - altStartI: 86862444
        altEndI: 86863605
        ord: 3
        altCdsStartI: 332
        altCdsEndI: 1492
        cigar: 1161M
      - altStartI: 86880661
        altEndI: 86880725
        ord: 2
        altCdsStartI: 268
        altCdsEndI: 331
        cigar: 64M
      - altStartI: 86903280
        altEndI: 86903448
        ord: 1
        altCdsStartI: 100
        altCdsEndI: 267
        cigar: 168M
      - altStartI: 86914027
        altEndI: 86914126
        altCdsStartI: 1
        altCdsEndI: 99
        cigar: 99M
  geneToTx:
  - geneId: HGNC:17705
    txIds:
    - NM_004261.5
    - NM_203341.3
seqDb:
  aliases:
  - NM_004261.5
  - NM_203341.3
  aliasesIdx:
  - 0
  - 1
  seqs:
  - AGAAACCTGCGGAAAATGGTAGCGATGGCGGCTGGGCCGAGTGGGTGTCTGGTGCCGGCGTTTGGGCTACGGTTGTTGTTGGCGACTGTGCTTCAAGCGGTGTCTGCTTTTGGGGCAGAGTTTTCATCGGAGGCATGCAGAGAGTTAGGCTTTTCTAGCAACTTGCTTTGCAGCTCTTGTGATCTTCTCGGACAGTTCAACCTGCTTCAGCTGGATCCTGATTGCAGAGGATGCTGTCAGGAGGAAGCACAATTTGAAACCAAAAAGCTGTATGCAGGAGCTATTCTTGAAGTTTGTGGATGAAAATTGGGAAGGTTCCCTCAAGTCCAAGCTTTTGTTAGGAGTGATAAACCCAAACTGTTCAGAGGACTGCAAATCAAGTATGTCCGTGGTTCAGACCCTGTATTAAAGCTTTTGGACGACAATGGGAACATTGCTGAAGAACTGAGCATTCTCAAATGGAACACAGACAGTGTAGAAGAATTCCTGAGTGAAAAGTTGGAACGCATATAAATCTTGCTTAAATTTTGTCCTATCCTTTTGTTACCTTATCAAATGAAATATTACAGCACCTAGAAAATAATTTAGTTTTGCTTGCTTCCATTGATCAGTCTTTTACTTGAGGCATTAAATATCTAATTAAATCGTGAAATGGCAGTATAGTCCATGATATCTAAGGAGTTGGCAAGCTTAACAAAACCCATTTTTTATAAATGTCCATCCTCCTGCATTTGTTGATACCACTAACAAAATGCTTTGTAACAGACTTGCGGTTAATTATGCAAATGATAGTTTGTGATAATTGGTCCAGTTTTACGAACAACAGATTTCTAAATTAGAGAGGTTAACAAGACAGATGATTACTATGCCTCATGTGCTGTGTGCTCTTTGAAAGGAATGACAGCAGACTACAAAGCAAATAAGATATACTGAGCCTCAACAGATTGCCTGCTCCTCAGAGTCTCTCCTATTTTTGTATTACCCAGCTTTCTTTTTAATACAAATGTTATTTATAGTTTACAATGAATGCACTGCATAAAAACTTTGTAGCTTCATTATTGTAAAACATATTCAAGATCCTACAGTAAGAGTGAAACATTCACAAAGATTTGCGTTAATGAAGACTACACAGAAAACCTTTCTAGGGATTTGTGTGGATCAGATACATACTTGGCAAATTTTTGAGTTTTACATTCTTACAGAAAAGTCCATTTAAAAGTGATCATTTGTAAGACCAAAATATAAATAAAAAGTTTCAAAAATCTATCTGAATTTGGAATTCTTCTGGTTTGTTCTTTCATGTTTAAAAATGATGTTTTTCAATGCATTTTTTTCATGTAAGCCCTTTTTTTAGCCAAAATGTAAAAATGGCTGTAATATTTAAAACTTATAACATCTTATTGTTGGTAATAGTGCTTTATATTTGTCTGATTTTATTTTTCAAAGTTTTTTCATTTATGAACACATTTTCATTGGTATATTATTTAAGGAATATCTCTTGATATAGAATTTTTATATTAAAAATGATTTTTCTTTGCTTAA
  - AGAAACCTGCGGAAAATGGTAGCGATGGCGGCTGGGCCGAGTGGGTGTCTGGTGCCGGCGTTTGGGCTACGGTTGTTGTTGGCGACTGTGCTTCAAGCGGTGTCTGCTTTTGGGGCAGAGTTTTCATCGGAGGCATGCAGAGAGTTAGGCTTTTCTAGCAACTTGCTTTGCAGCTCTTGTGATCTTCTCGGACAGTTCAACCTGCTTCAGCTGGATCCTGATTGCAGAGGATGCTGTCAGGAGGAAGCACAATTTGAAACCAAAAAGCTGTATGCAGGAGCTATTCTTGAAGTTTGTGGATGAAAATTGGGAAGGTTCCCTCAAGTCCAAGTATGTCCGTGGTTCAGACCCTGTATTAAAGCTTTTGGACGACAATGGGAACATTGCTGAAGAACTGAGCATTCTCAAATGGAACACAGACAGTGTAGAAGAATTCCTGAGTGAAAAGTTGGAACGCATATAAATCTTGCTTAAATTTTGTCCTATCCTTTTGTTACCTTATCAAATGAAATATTACAGCACCTAGAAAATAATTTAGTTTTGCTTGCTTCCATTGATCAGTCTTTTACTTGAGGCATTAAATATCTAATTAAATCGTGAAATGGCAGTATAGTCCATGATATCTAAGGAGTTGGCAAGCTTAACAAAACCCATTTTTTATAAATGTCCATCCTCCTGCATTTGTTGATACCACTAACAAAATGCTTTGTAACAGACTTGCGGTTAATTATGCAAATGATAGTTTGTGATAATTGGTCCAGTTTTACGAACAACAGATTTCTAAATTAGAGAGGTTAACAAGACAGATGATTACTATGCCTCATGTGCTGTGTGCTCTTTGAAAGGAATGACAGCAGACTACAAAGCAAATAAGATATACTGAGCCTCAACAGATTGCCTGCTCCTCAGAGTCTCTCCTATTTTTGTATTACCCAGCTTTCTTTTTAATACAAATGTTATTTATAGTTTACAATGAATGCACTGCATAAAAACTTTGTAGCTTCATTATTGTAAAACATATTCAAGATCCTACAGTAAGAGTGAAACATTCACAAAGATTTGCGTTAATGAAGACTACACAGAAAACCTTTCTAGGGATTTGTGTGGATCAGATACATACTTGGCAAATTTTTGAGTTTTACATTCTTACAGAAAAGTCCATTTAAAAGTGATCATTTGTAAGACCAAAATATAAATAAAAAGTTTCAAAAATCTATCTGAATTTGGAATTCTTCTGGTTTGTTCTTTCATGTTTAAAAATGATGTTTTTCAATGCATTTTTTTCATGTAAGCCCTTTTTTTAGCCAAAATGTAAAAATGGCTGTAATATTTAAAACTTATAACATCTTATTGTTGGTAATAGTGCTTTATATTTGTCTGATTTTATTTTTCAAAGTTTTTTCATTTATGAACACATTTTCATTGGTATATTATTTAAGGAATATCTCTTGATATAGAATTTTTATATTAAAAATGATTTTTCTTTGCTTAA
version: 0.0.0
genomeRelease: GRCh38

